jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -150,6 +150,14 @@ protected void navigateToPerspective(final String perspectiveId,
                              parameterValue);
         placeManager.goTo(request);
     }
+    
+    public void openView(final PlaceRequest request) {
+        placeManager.goTo(request);
+    }
+    
+    public void openView(final PlaceRequest request, final String parameterValue) {
+        placeManager.goTo(request);
```

Comment:
This method really looks odd. It does exactly the same thing as the one above, but for some reason takes two arguments.
If we can't avoid having this two-arguments method here, can we at least make it abstract and rely on overriding it wherever it is used?